releasenotes/notes/spans_can_opt_out_of_global_tags-3afc437e6b706b07.yaml

```
@@ -0,0 +1,10 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - Some spans can opt out of having global tags set.
```

We have to prefix entries for trace-agent with `APM: `.
```suggestion
  - APM: Some spans can opt out of having global tags set.
```